.github/workflows/test_simple.sh

```diff
@@ -3,4 +3,19 @@
 # fail on non-zero return code from a subprocess
 set -e
 
+echo $PATH
+# Test if grass executable is found
+which grass
+grass_exe=$(which grass)
+
+which python3
+python3 -c "import sys; print(sys.executable, sys.version)"
```

Comment:
There is now also https://github.com/OSGeo/grass/blob/main/.github/workflows/print_versions.sh (introduced in #1832) which is a separate step with `if: always()`. Having the versions here also un-syncs it with the `.bat` version.

Reply:
Updated to use .github/workflows/print_versions.sh